common/types/block.go

```diff
@@ -130,8 +130,10 @@ func (b *BlockHeader) AddView(id BlockID) {
 // produce the block signature.
 type MiniBlock struct {
 	BlockHeader
-	TxIDs  []TransactionID
-	ATXIDs []ATXID
+	TxIDs     []TransactionID
+	ATXIDs    []ATXID
+	ActiveSet *[]ATXID
+	RefBlock  *BlockID
```

Comment:
so this is the refblock just for the active set thing ? and it will be in addition to what we'll use as "base block" in tortoise right ?

Reply:
yes